tests/behat/src/CmsUiContext.php

```diff
@@ -156,6 +170,28 @@ public function stepIShouldNotSeeInCmsTree($text)
     }
 
     /**
+     * @When /^I should see "([^"]*)" in the list$/
```

Comment:
Can you please make sure the declaration matches the specifics? This fixture isn't useful for any lists, but exclusively "page list view".